pkgs/tools/misc/0x0/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, pkgs, xsel, curl, fetchFromGitLab, makeWrapper}:
+
+stdenv.mkDerivation rec {
+  name = "0x0-${version}";
+  version = "1.0";
```

Comment:
I wouldn't call that 1.0, to be honest; something date-based is probably more suitable.

Reply:
Thanks for the input, I've changed the version number to the date of the rev